csrc/ir/utils.cpp

```diff
@@ -250,9 +249,9 @@ Expr* transferDefinitionToNewOutputs(
   return mutator.mutateExprOutputsOnly(expr);
 }
 
-TensorView* rfactorHelper(
+TensorView* rFactorHelper(
```

Comment:
nitpicking: is `rfactor` not a word? It reads a bit strange.

Reply:
nvm, I see where you are going with this.

Reply:
Yeah, I think rFactor corresponds to reduction-factor, which is a compound word.

Reply:
^^^ yeah, but we are using `xxx_rfactor_yyy` in some variable names, which sorta blend them as a single word.

Not saying that I'm in favor of `xxx_r_factor_yyy`. :laughing: I'm happy with your clean up already.